### PR TITLE
added cost management documentation

### PR DESCRIPTION
**Pull request by FungCharlotte**

### CODE REVIEW

**Comment by cwooley-rh** [comment]
Would we want to segment out the Azure Cost mgmt best practices from AWS?

**Comment by FungCharlotte** [comment]
@cwooley-rh , you are right. I'll take out that section, and then do another documentation on how to add AWS cloud source to cost management. Thanks

**Review by cwooley-rh — APPROVED** [review]

File: docs/cost-management/README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+Cost management will begin polling Azure for cost data, which will appear on the cost management dashboard (console.redhat.com/openshift/cost-management/).
+
+
+## Managing your Costs
+
+After adding your Openshift Container Platform and Cloud Provider sources, Cost management will show cost data by 
+1. Source
+1. Cloud provider cost and usage related to running your OpenShift Container Platform clusters on their platform 
+
+> See the following video for a quick overview of Cost Management for OpenShift followed by a demo of the product
```
Comment (paulczar) [inline docs/cost-management/README.md:186]: this should be indented three times `   > See the ....`  in order to be inside the ordered list.

File: docs/cost-management/README.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+1. Return to **Sources** wizard after creating the export schedule and click **Next**. Review the source details 
+
+1. Click **Finish** to complete adding the Azure source to cost management
+
+Cost management will begin polling Azure for cost data, which will appear on the cost management dashboard (console.redhat.com/openshift/cost-management/).
+
+
+## Managing your Costs
+
+After adding your Openshift Container Platform and Cloud Provider sources, Cost management will show cost data by
```
Comment (paulczar) [inline docs/cost-management/README.md:182]: a space between this and the ordered list.

**Review by paulczar — CHANGES_REQUESTED** [review]

**Pull request merged**